tests/t_server_null_stress.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Run this stress test as root to avoid sudo authorization from timing out.
+
+count=1
+
+. ./t_server_null_default.rc
+
+export pid_files=""
+for SUF in $TEST_SERVER_LIST
+do
+    eval server_name=\"\$SERVER_NAME_$SUF\"
+    pid_files="${pid_files} ${srcdir}/${server_name}.pid"
+done
+
+LOG_BASEDIR="make-check"
+mkdir -p "${LOG_BASEDIR}"
+
+while [ $count -lt 100 ]; do
+    count=$(( count + 1 ))
+    make check > /dev/null 2>&1
```

Comment:
```suggestion
    make check  TESTS=t_server_null.sh SUBDIRS= > /dev/null 2>&1
```
That would only run the interesting test, not all of them.

Reply:
Make a lot of sense :+1: